### PR TITLE
add npm scripts (postinstall), update readme file

### PR DESCRIPTION
**Pull request by aelbore**
* add npm scripts to install both frontend and backend
* add npm scripts to run both frontend and backend
* update readme file

the dev will run scripts in the root folder no need to "cd frontend or backend" in order to run

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,17 @@ This is a student-teacher matching application made for the MAD WISH initiative.
...
-## Getting started
+Getting Started
+------------
 Code and setup instructions for frontend and backend can be accessed in their respective subdirectories.
 
-TODO: global setup instructions
\ No newline at end of file
+* Install dependencies
+  ```
+    git clone https://github.com/therealyingtong/irrc-madwish.git
```
Comment (therealyingtong) [inline README.md:13]:
```suggestion
    git clone https://github.com/TechLadies/irrc-madwish.git
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "scripts": {
+    "install.frontend": "yarn --cwd frontend install",
+    "install.backend": "yarn --cwd backend install",
+    "postinstall": "yarn clean && concurrently \"yarn install.frontend\" \"yarn install.backend\"",
+    "serve": "concurrently \"yarn --cwd frontend serve\"",
+    "clean": "concurrently \"rm -rf frontend/node_modules\"  \"rm -rf backend/node_modules\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aelbore/irrc-madwish.git"
```
Comment (therealyingtong) [inline package.json:15]:
```suggestion
    "url": "git+https://github.com/TechLadies/irrc-madwish.git"
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/aelbore/irrc-madwish/issues"
```
Comment (therealyingtong) [inline package.json:21]:
```suggestion
    "url": "https://github.com/TechLadies/irrc-madwish/issues"
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "homepage": "https://github.com/aelbore/irrc-madwish#readme",
```
Comment (therealyingtong) [inline package.json:23]:
```suggestion
  "homepage": "https://github.com/TechLadies/irrc-madwish#readme",
```

**Review by therealyingtong — APPROVED** [review]
change URLs to `TechLadies/irrc-madwish`

**Comment by aelbore** [comment]
@therealyingtong looks good

**Pull request merged**